music_assistant/providers/airplay/player.py

```diff
@@ -271,25 +291,33 @@ async def play_media(self, media: PlayerMedia) -> None:
             )
 
         # if an existing stream session is running, we could replace it with the new stream
-        if self.raop_stream and self.raop_stream.running:
+        if self.stream and self.stream.running:
             # check if we need to replace the stream
-            if self.raop_stream.prevent_playback:
+            if self.stream.prevent_playback:
                 # player is in prevent playback mode, we need to stop the stream
                 await self.stop()
             else:
-                await self.raop_stream.session.replace_stream(audio_source)
+                await self.stream.session.replace_stream(audio_source)
                 return
 
-        # setup RaopStreamSession for player (and its sync childs if any)
+        # setup StreamSession for player (and its sync childs if any)
         sync_clients = self._get_sync_clients()
         provider = cast("AirPlayProvider", self.provider)
-        raop_stream_session = RaopStreamSession(provider, sync_clients, input_format, audio_source)
-        await raop_stream_session.start()
+        if self.mass.config.get_raw_player_config_value(self.player_id, CONF_AIRPLAY_VERSION) == 2:
+            ap2_stream_session = AirPlay2StreamSession(
```

Comment:
In a follow-up PR we should aim on using a single session regardless the ap version